src/types.ts

```diff
@@ -10,9 +10,9 @@ export interface CliGlobalOptions {
 
 // TODO: Try to have all properties defined
 interface ListrContextBuildItem {
-  releaseDir?: string;
+  variant: string | null;
```

Comment:
same as below. What means variant? what is the intention behind the scenes allowing null

Reply:
Variant meaning is explained here: https://github.com/dappnode/DAppNodeSDK/pull/443#discussion_r1679148654

It can be `null` because legacy format packages (with full compose and manifest files in the root of the repo) have no `variant` so it makes sense that it is explicitly `null`